website/client/src/components/shops/buyModal.vue

```diff
@@ -164,6 +165,18 @@
       ></span>
       <span class="limitedString">{{ limitedString }}</span>
     </div>
+    <div
+      v-if="!itemAvailable"
+      class="notAvailable"
+    >
+      <span
+        class="svg-icon inline icon-16 clock-icon"
```

Comment:
can we add a `v-once` directive here to make sure the icon is only rendered once? helps with performances